src/lightbox/tp-lightbox.ts

```diff
@@ -482,4 +490,48 @@ export class TPLightboxElement extends HTMLElement {
 			}
 		}
 	}
+
+	/**
+	 * Set the current slide index.
+	 *
+	 * @param {number} index Slide index.
+	 */
+	setCurrentSlide( index: number ): void {
```

Comment:
@nirav7707 Looking more closely, looks like we already have a setter: https://github.com/Travelopia/web-components/blob/1ce35860648b7b651d36765e74f71196217cb720/src/lightbox/tp-lightbox.ts#L162

Reply:
Update done!